keyboards/lyra/rev1/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "lyra", 
+    "url": "", 
+    "maintainer": "malevolti", 
+    "width": 15, 
+    "height": 6.3999999999999995, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"x":4, "y":0}, {"x":10, "y":0}, {"x":3, "y":0.09999999999999998}, {"x":5, "y":0.09999999999999998}, {"x":9, "y":0.09999999999999998}, {"x":11, "y":0.09999999999999998}, {"x":6, "y":0.25}, {"x":8, "y":0.25}, {"x":0, "y":0.30000000000000004}, {"x":2, "y":0.30000000000000004}, {"x":12, "y":0.30000000000000004}, {"x":14, "y":0.30000000000000004}, {"x":1, "y":0.4}, {"x":13, "y":0.4}, {"x":10, "y":1.0}, {"x":4, "y":1}, {"x":3, "y":1.0999999999999999}, {"x":5, "y":1.0999999999999999}, {"x":9, "y":1.0999999999999999}, {"x":11, "y":1.0999999999999999}, {"x":6, "y":1.25}, {"x":8, "y":1.25}, {"x":0, "y":1.3}, {"x":2, "y":1.3}, {"x":12, "y":1.3}, {"x":14, "y":1.3}, {"x":1, "y":1.4}, {"x":13, "y":1.4}, {"x":4, "y":2.0}, {"x":10, "y":2.0}, {"x":3, "y":2.1}, {"x":5, "y":2.1}, {"x":9, "y":2.1}, {"x":11, "y":2.1}, {"x":6, "y":2.25}, {"x":8, "y":2.25}, {"x":0, "y":2.3}, {"x":2, "y":2.3}, {"x":12, "y":2.3}, {"x":14, "y":2.3}, {"x":1, "y":2.4}, {"x":13, "y":2.4}, {"x":4, "y":3.0}, {"x":10, "y":3.0}, {"x":3, "y":3.1}, {"x":5, "y":3.1}, {"x":9, "y":3.1}, {"x":11, "y":3.1}, {"x":6, "y":3.25}, {"x":8, "y":3.25}, {"x":0, "y":3.3}, {"x":2, "y":3.3}, {"x":12, "y":3.3}, {"x":14, "y":3.3}, {"x":1, "y":3.4}, {"x":13, "y":3.4}, {"x":4, "y":4.0}, {"x":10, "y":4.0}, {"x":3, "y":4.1}, {"x":5, "y":4.1}, {"x":9, "y":4.1}, {"x":11, "y":4.1}, {"x":0, "y":4.3}, {"x":2, "y":4.3}, {"x":12, "y":4.3}, {"x":14, "y":4.3}, {"x":1, "y":4.3999999999999995}, {"x":13, "y":4.3999999999999995}, {"x":6.25, "y":4.5}, {"x":7.75, "y":4.5}]
+        }
+    }
+}
```

Comment:
```suggestion
{
    "keyboard_name": "lyra",
    "url": "",
    "maintainer": "malevolti",
    "width": 15,
    "height": 5.5,
    "layouts": {
        "LAYOUT": {
            "layout": [
                {"x": 0, "y": 0.3},
                {"x": 1, "y": 0.4},
                {"x": 2, "y": 0.3},
                {"x": 3, "y": 0.1},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0.1},
                {"x": 6, "y": 0.25},

                {"x": 8, "y": 0.25},
                {"x": 9, "y": 0.1},
                {"x": 10, "y": 0},
                {"x": 11, "y": 0.1},
                {"x": 12, "y": 0.3},
                {"x": 13, "y": 0.4},
                {"x": 14, "y": 0.3},

                {"x": 0, "y": 1.3},
                {"x": 1, "y": 1.4},
                {"x": 2, "y": 1.3},
                {"x": 3, "y": 1.1},
                {"x": 4, "y": 1},
                {"x": 5, "y": 1.1},
                {"x": 6, "y": 1.25},

                {"x": 8, "y": 1.25},
                {"x": 9, "y": 1.1},
                {"x": 10, "y": 1},
                {"x": 11, "y": 1.1},
                {"x": 12, "y": 1.3},
                {"x": 13, "y": 1.4},
                {"x": 14, "y": 1.3},

                {"x": 0, "y": 2.3},
                {"x": 1, "y": 2.4},
                {"x": 2, "y": 2.3},
                {"x": 3, "y": 2.1},
                {"x": 4, "y": 2},
                {"x": 5, "y": 2.1},
                {"x": 6, "y": 2.25},

                {"x": 8, "y": 2.25},
                {"x": 9, "y": 2.1},
                {"x": 10, "y": 2},
                {"x": 11, "y": 2.1},
                {"x": 12, "y": 2.3},
                {"x": 13, "y": 2.4},
                {"x": 14, "y": 2.3},

                {"x": 0, "y": 3.3},
                {"x": 1, "y": 3.4},
                {"x": 2, "y": 3.3},
                {"x": 3, "y": 3.1},
                {"x": 4, "y": 3},
                {"x": 5, "y": 3.1},
                {"x": 6, "y": 3.25},

                {"x": 8, "y": 3.25},
                {"x": 9, "y": 3.1},
                {"x": 10, "y": 3},
                {"x": 11, "y": 3.1},
                {"x": 12, "y": 3.3},
                {"x": 13, "y": 3.4},
                {"x": 14, "y": 3.3},

                {"x": 0, "y": 4.3},
                {"x": 1, "y": 4.4},
                {"x": 2, "y": 4.3},
                {"x": 3, "y": 4.1},
                {"x": 4, "y": 4},
                {"x": 5, "y": 4.1},
                {"x": 6.25, "y": 4.5},

                {"x": 7.75, "y": 4.5},
                {"x": 9, "y": 4.1},
                {"x": 10, "y": 4},
                {"x": 11, "y": 4.1},
                {"x": 12, "y": 4.3},
                {"x": 13, "y": 4.4},
                {"x": 14, "y": 4.3}
            ]
        }
    }
}
```
There were still some rounding errors, and the ordering was incorrect anyway. I've fixed them here and cleaned it up a bit.